docs/source/blog/Introducing-1-0-beta.md

```diff
@@ -133,6 +133,18 @@ dotnet fable add fable-powerpack@next fable-react@next fable-elmish@next fable-e
 
 > At the moment Fable cannot resolve transitive dependencies so it's necessary to list them all.
 
+#### Important Note
+
+When adding new dependencies and making changes to the project file, it is often necessary to run the following commands again.
+
+```shell 
+ dotnet restore
+ dotnet build
+ dotnet fable npm-run build
```

Comment:
In theory, with latest Ionide & FSharp.Net.Sdk you shouldn't need to run `dotnet build`. I need to check. Also, maybe we can take `dotnet fable npm-run build` out of this list as it's not directly related with intellisense and users may want to use another npm-script (e.g., `start`).